mon-pix/app/components/autonomous-course/landing-page-start-block.gjs

```diff
@@ -42,33 +57,16 @@ export default class LandingPageStartBlock extends Component {
             id="autonomous-course-sign-in-button"
             class="sign-in-button"
             type="button"
-            {{on "click" this.redirectToSigninIfUserIsAnonymous}}
+            {{on "click" this.redirectToSignin}}
           >
             {{t "pages.autonomous-course.landing-page.actions.sign-in"}}
           </button>
         </div>
       {{/if}}
+
       <p class="autonomous-course-landing-page-start-block__informations">
         {{t "pages.autonomous-course.landing-page.texts.legal-informations" htmlSafe=true}}
       </p>
     </section>
   </template>
-  @service session;
-  @service router;
-
-  get isUserConnected() {
-    return this.session.isAuthenticated;
-  }
-
-  @action
-  async redirectToSigninIfUserIsAnonymous(event) {
-    event.preventDefault();
-
-    if (this.isUserConnected) {
-      this.router.transitionTo('authenticated');
-    } else {
-      const transition = this.args.startCampaignParticipation();
-      this.session.requireAuthenticationAndApprovedTermsOfService(transition);
-    }
```

Comment:
Le `if (this.isUserConnected)` a été supprimé de cette fonction car elle ne peut jamais être déclenchée si l'utilisateur est connecté (voir le if dans template)